apps/meteor/tests/e2e/display-mode.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { Users } from './fixtures/userStates';
+import { HomeChannel } from './page-objects';
+import { createTargetChannel, deleteChannel, sendTargetChannelMessage } from './utils';
+import { test, expect } from './utils/test';
+
+test.use({ storageState: Users.admin.state });
+let sentMessage: string;
+let channelName: string;
+test.describe.serial('display settings', () => {
+	let poHomeChannel: HomeChannel;
+
+	test.beforeAll(async ({ api }) => {
+		channelName = await createTargetChannel(api, { members: ['user1', 'user2'] });
+	});
+
+	test.beforeEach(async ({ page }) => {
+		poHomeChannel = new HomeChannel(page);
+
+		await page.goto('/home');
+	});
+
+	test.afterAll(async ({ api }) => {
+		await deleteChannel(api, channelName);
```

Comment:
I think the preference should be reseted,  even though  one step resets partially the state, but if fails next steps would inherit non default settings